src/main/java/com/cloudbees/hudson/plugins/folder/computed/FolderCron.java

```diff
@@ -114,11 +114,7 @@ public void doRun() {
      * @param cal the date to check for.
      */
     public void checkTriggers(final Calendar cal) {
-        Jenkins inst = Jenkins.getInstance();
-        if (inst == null) {
-            return;
-        }
-        for (ComputedFolder<?> p : inst.getAllItems(ComputedFolder.class)) {
+        for (ComputedFolder<?> p : Jenkins.getInstance().getAllItems(ComputedFolder.class)) {
```

Comment:
Don’t we have `allItems` to save sorting the list?

Reply:
Yes, we could apply it here too. I did not because it was not the subject of the PR and it did not at first blush seem like terribly performance-sensitive code. Looking more closely I see that this will be run every minute even if triggers are disabled or infrequent, and I see that you changed the same thing in `Trigger.checkTriggers`, so it probably does make sense.